docs/sdkx-unity/tracking-android.mdx

```diff
@@ -196,6 +196,17 @@ On Custom Issue Fields keys & compatibility
 
 </Admonition>
 
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, passing of formatted strings is not supported. It is recommended to pass data only in a simple String format.
+
+For example - 
+
+* "{\"key\":\"value\"}" - ** JSON String, This is not supported **
+* "key - value" - ** Simple string, This is supported **
```

Comment:
Same for this

Reply:
Its just string formats, Do we need code snippets here?